__tests__/app.test.js

```diff
@@ -266,6 +266,42 @@ describe("app", () => {
     });
   });
 
+  describe("DELETE /api/comments/:comment_id", () => {
+    test("204: returns 204 and a blank response", () => {
+      return request(app)
+        .delete("/api/comments/1")
+        .expect(204)
+        .then(({ body }) => {
+          expect(body).toEqual({});
```

Comment:
DELETE doesn't return anything so no need to expect body to be empty. Can get rid of the .then block